src/Universum/Container/Class.hs

```diff
@@ -596,6 +598,11 @@ instance Container (Vector a)
 
 -- TODO: I should put different strings for different versions but I'm too lazy to do it...
 
+-- | Similar to 'foldl'' but takes a function with its arguments flipped.
```

Comment:
Could you add some examples to this function? Something like: `flipfoldl (/) 5 [2,3] :: Rational` just to see how things are evaluated.